# see: https://github.com/openzfs/zfs/issues/11357
# see: https://github.com/openzfs/zfs/issues/14555
#
# note... I take no responsibility for whatever legal license nonsense
#         you may implicate yourself in by using this, if that should happen
#
--- src/META
+++ src/META
@@ -4,7 +4,7 @@
 Version:       2.1.11
 Release:      1
 Release-Tags:  relext
-License:       CDDL
+License:       GPL
 Author:        OpenZFS
 Linux-Maximum: 6.2
 Linux-Minimum: 3.10
